packages/hub/indexing/document-context.js

```diff
@@ -105,7 +102,7 @@ module.exports = class DocumentContext {
 
   async _buildAttribute(field, value, pristineDocOut, searchDocOut) {
     // Write our value into the search doc
-    let esName = await this._logicalFieldToES(field.id);
+    let esName = field.id;
```

Comment:
Now `esName` as a variable isn't really needed either. You can replace it everywhere with `field.id`.